roles/openshift_node_group/tasks/sync.yml

```diff
@@ -70,5 +70,36 @@
     - __status_of_sync_ds.results.results[0].status.desiredNumberScheduled is defined
     - __status_of_sync_ds.results.results[0].status.numberAvailable == __status_of_sync_ds.results.results[0].status.desiredNumberScheduled
   retries: 60
-  delay: 30
-  failed_when: false
+  delay: 10
+
+- name: Wait for sync DS to set annotations on all nodes
+  oc_obj:
+    state: list
+    kind: node
+    selector: ""
+  register: node_status
+  until:
+    - node_status.results is defined
+    - node_status.results.results is defined
+    - node_status.results.results | length > 0
+    - node_status.results.results[0]['items']
+     | map(attribute='metadata.annotations') | map('list') | flatten
```

Comment:
This would collect a list of annotations for each node. The list is expected to have duplicate items so on line 87 we'll wait until a number of nodes present would be equal to a number of annotated nodes.

At this point we don't need to check the checksum of the config as sync pods haven't updated it yet

Reply:
@vrutkovs curious: i suspect you don't need `map('list')` and you can do it
`- node_status.results.results[0]['items']
     | map(attribute='metadata.annotations') | list | flatten`